src/main/java/com/extremelyd1/listener/ItemListener.java

```diff
@@ -53,4 +56,55 @@ public void onPlayerDropItem(PlayerDropItemEvent e) {
             e.setCancelled(true);
         }
     }
+
+    /**
+     * Caters for collecting milk, water or lava
+     *
+     * @param e - PlayerBucketFillEvent
```

Comment:
I don't have javadoc written for each method (especially not for event handler methods), but if you happen to write it, please also fill in the `@param` section.

Reply:
addressed in recent commit, I just removed the java docs if they're not required.